fabtests/common/shared.c

```diff
@@ -3059,6 +3064,25 @@ void ft_parse_hmem_opts(int op, char *optarg, struct ft_opts *opts)
 	}
 }
 
+void parse_opts_range (char* optarg)
+{
+       int start, inc, end;
+       int i, range_test_cnt;
+       sscanf (optarg, "r:%d,%d,%d", &start, &inc, &end);
+
+       /* populate the array */
+       range_test_cnt = (end - start) / inc + 1;
+       TEST_CNT = range_test_cnt;
+       range_test_size = malloc (sizeof (struct test_size_param) * 
```

Comment:
remove space after function name.  it would be safer here to use calloc() and avoid the initialization to 0 below.  As a general rule, sizeof() should be paired with a variable name, not a structure definition.  E.g.
```
range_test_size = calloc(test_cnt, sizeof(*range_test_size));
```
Add a check for allocation failure